zaf/zaf/messages/messagebus.py

```diff
@@ -267,7 +267,8 @@ def send_request(self, message_id, receiver_endpoint_id=None, entity=None, data=
 
         :param message_id: The MessageId of the request that should be sent
         :param receiver_endpoint_id: The EndpointId of the recipients of the messages
-        :para data: The message data
+        :param entity: Only send request to certain entity at endpoint.
```

Comment:
replace "certain" with "specified". Otherwise good update.
New review after the above change is not needed - for my sake.